ql/src/java/org/apache/hadoop/hive/ql/exec/ReplCopyTask.java

```diff
@@ -159,7 +159,7 @@ protected int execute(DriverContext driverContext) {
         if (!rwork.getListFilesOnOutputBehaviour(oneSrc)){
```

Comment:
OK, agreed! But, I believe, ReplCopyTask won't go away in case of REPL LOAD flow. I'm just concerned that, this unused code shouldn't remain forever.